kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/SchemaManager.java

```diff
@@ -336,13 +340,14 @@ com.google.cloud.bigquery.Schema getAndValidateProposedSchema(
    */
   private List<com.google.cloud.bigquery.Schema> getSchemasList(TableId table, List<SinkRecord> records) {
     List<com.google.cloud.bigquery.Schema> bigQuerySchemas = new ArrayList<>();
-    Optional.ofNullable(readTableSchema(table)).ifPresent(bigQuerySchemas::add);
+    Optional<com.google.cloud.bigquery.Schema> existingSchema = Optional.ofNullable(readTableSchema(table));
+    existingSchema.ifPresent(bigQuerySchemas::add);
     for (SinkRecord record : records) {
       Schema kafkaValueSchema = schemaRetriever.retrieveValueSchema(record);
       if (kafkaValueSchema == null) {
         continue;
       }
-      bigQuerySchemas.add(convertRecordSchema(record));
+      bigQuerySchemas.add(convertRecordSchema(record, existingSchema.orElse(null)));
```

Comment:
Why do we need this change here? Isn't `getUnionizedSchema` already covering the schema unionization with biogquery existing schema?